src/Observables.jl

```diff
@@ -95,13 +95,14 @@ function discretizationlogder(B, D::PwMap; degree = 7)
         ind_X1 = Int64(floor(length(B)*br.X[1].lo))+1
         ind_X2 = min(Int64(floor(length(B)*br.X[2].lo))+1, length(B))    
         dom = hull(br.X[1], br.X[2])
+        fprime = derivative(br.f)
```

Comment:
I like this interface, dispatching on the type of the function is a really good idea.